README.md

```diff
@@ -476,15 +477,15 @@ Rough estimation of the required resources for ingestion path:
 
 * Network usage: outbound traffic is negligible. Ingress traffic is ~100 bytes per ingested data point via
   [Prometheus remote_write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
-  The actual ingress bandwitdh usage depends on the average number of labels per ingested metric and the average size
-  of label values. Higher number of per-metric lables and longer label values mean higher ingress bandwidth.
+  The actual ingress bandwidth usage depends on the average number of labels per ingested metric and the average size
+  of label values. The higher number of per-metric labels and longer label values mean higher ingress bandwidth.
```

Comment:
`the` transforms construction into comparison like `the more ... the more` but without second part. It should be `the higher number ..., the higher ingress...`